2024/01/17 12:10:06 [error] 195#195: *24 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8" failed (2: No such file or directory), client: 3.145.35.29, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8"
2024/01/17 12:10:06 [error] 195#195: *25 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8" failed (2: No such file or directory), client: 18.237.143.92, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8"
2024/01/17 12:10:07 [error] 195#195: *26 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8" failed (2: No such file or directory), client: 23.178.112.200, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/5xNgoxzOdB6EHG63sy7Bp-jqqnqlCaQJd14HeKB4oA8"
2024/01/17 13:12:33 [error] 195#195: *92 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0" failed (2: No such file or directory), client: 3.145.35.29, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0"
2024/01/17 13:12:33 [error] 195#195: *93 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0" failed (2: No such file or directory), client: 34.220.103.195, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0"
2024/01/17 13:12:35 [error] 195#195: *94 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0" failed (2: No such file or directory), client: 23.178.112.106, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/Y_TGa16FYRLgODGyGYVTDBHRPYVPjbTmyPO9dAhckO0"
2024/01/17 14:11:30 [error] 195#195: *150 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48" failed (2: No such file or directory), client: 3.145.35.29, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48"
2024/01/17 14:11:30 [error] 195#195: *152 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48" failed (2: No such file or directory), client: 18.237.143.92, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48"
2024/01/17 14:11:30 [error] 195#195: *153 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48" failed (2: No such file or directory), client: 23.178.112.101, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48 HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/8Y4fi6NODSjhRxFROcxos78VtZnl471HhjW92k4ds48"
2024/01/17 15:13:25 [error] 195#195: *314 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis" failed (2: No such file or directory), client: 34.220.103.195, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis"
2024/01/17 15:13:25 [error] 195#195: *313 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis" failed (2: No such file or directory), client: 3.145.35.29, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis"
2024/01/17 15:13:26 [error] 195#195: *315 open() "/data/letsencrypt-acme-challenge/.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis" failed (2: No such file or directory), client: 23.178.112.207, server: nextcloud.hanisntsolo.online, request: "GET /.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis HTTP/1.1", host: "nextcloud.hanisntsolo.online", referrer: "http://nextcloud.hanisntsolo.online/.well-known/acme-challenge/y-cG7mfG9_rA4xwUB3DRSw8FuWCdYxV5Kj05vdh1yis"
